src/ContactPeople/ContactPeopleList.js

```diff
@@ -89,7 +89,10 @@ const ContactPeopleList = ({ fetchContacts, fields, contactsMap, orgId, categori
   const anchoredRowFormatter = (row) => (
     <div role="listitem" key={`row-${row.rowIndex}`}>
       <Link
-        to={getContactsUrl(orgId, row.rowData.id)}
+        to={{
+          pathname: getContactsUrl(orgId, row.rowData.id),
+          state: { canGoBack: true },
```

Comment:
tbd

Reply:
it's not safe to use, I've encountered the situation when it doesn't work, something did reset of the state after first render